code/src/MessageList.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react'
+import { LikeButton } from "./LikeButton"
+import moment from "moment"
+import './message-list.css'
+
+export const MessageList = () => {
```

Comment:
I'm thinking this maybe is more of a feed rather than a list?
```suggestion
export const Feed = () => {
```